src/services/codefixes/convertToTypeOnlyImport.ts

```diff
@@ -104,7 +106,11 @@ function canConvertImportDeclarationForSpecifier(specifier: ImportSpecifier, sou
     const checker = program.getTypeChecker();
     for (const specifier of nonTypeOnlySpecifiers) {
         const isUsedAsValue = FindAllReferences.Core.eachSymbolReferenceInFile(specifier.name, checker, sourceFile, usage => {
-            return !isValidTypeOnlyAliasUseSite(usage);
+            if (isValidTypeOnlyAliasUseSite(usage)) {
+                return true;
+            }
+            const symbol = checker.getSymbolAtLocation(usage);
+            return !!(symbol && skipAlias(symbol, checker).flags & SymbolFlags.Value);

```

Comment:
You should also check whether the alias’s value meaning resolves through a type-only import/export already, via `checker.getTypeOnlyAliasDeclaration(symbol, SymbolFlags.Value)` (you’ll need to update the signature on `TypeChecker`). If that call returns something, then this value reference isn’t really a value reference.

Reply:
Oke, I’ve changed